packages/backend/src/index.ts

```diff
@@ -127,14 +127,22 @@ export function initIocContainer(
       replica_addresses: config.tigerbeetleReplicaAddresses
     })
   })
-  container.singleton('openApi', async () => {
+  container.singleton('resourceServerOpenApi', async () => {
     const resourceServerSpec = await createOpenAPI(
       path.resolve(__dirname, './openapi/resource-server.yaml')
     )
     return {
       resourceServerSpec
     }
   })
+  container.singleton('walletAddressServerOpenApi', async () => {
+    const walletAddressServerSpec = await createOpenAPI(
+      path.resolve(__dirname, './openapi/wallet-address-server.yaml')
+    )
+    return {
+      walletAddressServerSpec
+    }
+  })
```

Comment:
I'm going to change that back to just `openApi` container that returns both specs. I don't know what I was thinking...